### ISSUE TITLE
ETQ instructeur / administrateur je souhaite avoir un espace organisé pour mes procédures

### ISSUE THREAD

**Issue by ErwanLHH** [betagouv/demarches-simplifiees.fr#4559]
Voici les captures d'écran de mon espace instructeur
![image](https://user-images.githubusercontent.com/49941223/69353302-96954900-0c7e-11ea-9146-3ffee5cdd564.png)
![image](https://user-images.githubusercontent.com/49941223/69353365-af056380-0c7e-11ea-83a3-e92aa1d3d76b.png)
![image](https://user-images.githubusercontent.com/49941223/69353416-bf1d4300-0c7e-11ea-8f96-37fd7c0c6d0e.png)

Certes, j'ai un usage un peu particulier et intensif de DS ;-) mais une interface plus organisée de mon espace instructeur serait grandement appréciable.

Solution 1 : des onglets pour les procédures selon leur état test/publiées/archivées plutôt que toutes empilées dans un même écran

Solution 2 : pouvoir organiser l'affichage avec des tris selon différents critères (état, date, nom...)

Solution 2+1

Solution 3 pouvoir organiser l'affichage rémanent des procédures comme pour les dossiers des instructeurs par exemple en masquant les procédures via une coche

**Comment by emsnytech** [comment]
OUi, c'est une issues que j'ai faite remonter il y a plus d'un an.
Un onglet par type de procédure, puis aussi en effet, les option tri etc
